nvml/requirements.in

```diff
@@ -1,2 +1,2 @@
-pynvml==8.0.4
+pynvml==11.4.1
```

Comment:
All the metric names are the same in this major version bump?

Reply:
We have been using pynvml 11.4.1 for months now. I can't remember why it was needed, perhaps v8 didn't support the latest hardware?

Reply:
Bump @cep21

Reply:
Ok to resolve this and merge.  It's unlikely the metric names changed for the API if it's working for him today.